.travis.yml

```diff
@@ -90,6 +90,7 @@ env:
     - "TESTLANG=V"
     - "TESTLANG=Vala"
     - "TESTLANG=VB"
+    - "TESTLANG=Mumps"
```

Comment:
Could you put this after `Lua`? Keeping it alphabetical.

Reply:
Done